### DISCUSSION TITLE
Password Database Reminder 80days

### DISCUSSION THREAD

**Discussion by Xtrato01** [YetiForceCompany/YetiForceCRM#15202] [How to]
Good day/afternoon/morning . 
 
 I have a question , how can i set an for 80 DAYS reminder since "last modified' or "created" password from the "password database" module.

 I found the workflow , but im not certain how to fully implement it as it involves some advanced variables i believe.

 This is what i have so far;
 
![image](https://user-images.githubusercontent.com/90271495/134047322-518f034a-353d-4c6d-b523-24c4f8b50f62.png)

![image](https://user-images.githubusercontent.com/90271495/134047992-84c953a2-db32-4069-b399-2149edcb4350.png)

Im stuck here as im not sure what would trigger a notice that this was "MODIFIED 80 DAYS" ago..

any help is welcomed . Thanks

**Comment by rskrzypczak** [comment]
You are not able to fully implement what you want with workflow.
The easiest and most effective way to do this is with a new cron action. However, this requires some development work as you are not able to get it with the currently available GUI tools.

**Reply by Xtrato01** [reply]
how would i go about that? and hours time ?
